src/main/java/uk/gov/pay/directdebit/tokens/api/TokenResponse.java

```diff
@@ -30,22 +27,24 @@
     @JsonProperty("mandate_reference")
     private String mandateReference;
 
+    @JsonProperty("mandate_type")
```

Comment:
What are the implications of this change?

Reply:
Currently frontend is not using this property (it is just parsed) here - https://github.com/alphagov/pay-direct-debit-frontend/blob/b3f29b12228874c7d363c964ed995e16d8ab64e3/common/clients/connector-client.js#L43
and
https://github.com/alphagov/pay-direct-debit-frontend/blob/b3f29b12228874c7d363c964ed995e16d8ab64e3/common/classes/Mandate.class.js#L14
so we renamed it to be consistent with:
- https://github.com/alphagov/pay-direct-debit-connector/blob/master/src/main/java/uk/gov/pay/directdebit/mandate/api/CreateMandateResponse.java#L24
- https://github.com/alphagov/pay-direct-debit-connector/blob/master/src/main/java/uk/gov/pay/directdebit/mandate/api/DirectDebitInfoFrontendResponse.java#L123
- https://github.com/alphagov/pay-direct-debit-connector/blob/master/src/main/java/uk/gov/pay/directdebit/mandate/api/GetMandateResponse.java#L20

and in our next frontend PR we are going to use the new property

Reply:
OK, if it’s not been used yet, that’s fine